content/blog/2024/announcing-de-shaw-as-silver-sponsor.md

```diff
@@ -0,0 +1,28 @@
+Title: Announcing our next Silver Sponsor - D. E. Shaw India
+Date: 2024-09-15 17:00
+Category: 2024
+Slug: sponsor-announcement-de-shaw
+Summary: We are excited to announce **D. E. Shaw India** as our next Silver Sponsor for PyCon India 2024.
+
+We are excited to announce **D. E. Shaw India** as our next Silver Sponsor for PyCon India 2024.
+
+<br>
+<div class="text-center">
+  <a href="https://deshawindia.com" target="_blank" style="border: none; text-decoration: none;">
+    <img src="{static}/images/2024/sponsor-deshaw.jpg" alt="Bloomberg" class="img-fluid responsive-image">
```

Comment:
Should be DE Shaw.

Reply:
@Sc0rpi0n101 alt="Bloomberg" should be there